### ISSUE TITLE
Awesome, Async support please!

### ISSUE THREAD

**Issue by TomYeoman** [binarykitchen/find-remove#8]
Great stuff, love cleaning up my all temp files of x age with a 1 liner :)

Async support would be great, but otherwise one happy user here.

**Comment by binarykitchen** [comment]
thanks mate. yeah, i know. can do when i have some free time. priorities and other works you know.

but tell me, are you asking this because you have performance issues?

**Comment by TomYeoman** [comment]
Yeah no worries mate, I might take a look at implementing this myself when I've got a minute. Good to see the project is still active.

I'm planning on using this on a large internal web application which I develop (100k+ daily users) and whilst I haven't ran into any performance related issues yet I just wanted to see if it was still in the pipeline.

**Comment by binarykitchen** [comment]
i see, sweet as. PRs welcome. or i ll do it in few months. or can do it now for a little bounty ;)

**Comment by steebchen** [comment]
+1 – I was installing this package when I realized that everything is sync. In a production server where a lot of things happen simultaneously, it's unusable.

**Comment by binarykitchen** [comment]
yeah i know and agree - again, PRs welcome

**Comment by davidlanger** [comment]
I created #13 but ci fails b/c node 0.10 is installed

**Comment by binarykitchen** [comment]
Hello guys, it's been ages. I've ported this cool package to TS, should be easier to implement this request now. PRs welcome ;)
